README.md

```diff
@@ -315,30 +315,30 @@ man in the middle (MITM) attacks.
 You need to provide your own, signed private key to avoid this kind of attacks (and make
 sure clients connect with verify-ca or verify-full sslmode).
 
-The following is an example Dockerfile that sets up a container with custom ssl private key and certificate:
+The following is an example docker-compose.yml that sets up a container with custom ssl private key and certificate:
+If you need to use your own ssl certificates you should mount them into 
 
-```
-FROM kartoza/postgis:11.0-2.5
-
-ADD ssl_cert.pem /etc/ssl/certs/ssl_cert.pem
-ADD localhost_ssl_key.pem /etc/ssl/private/ssl_key.pem
-
-RUN chmod 400 /etc/ssl/private/ssl_key.pem
-```
-
-And a docker-compose.yml to initialize with this configuration:
+**/configs/ssl_cert.pem** 
 
 ```
-services:
-  postgres:
-    build:
-      dockerfile: Dockerfile
-      context: ssl_secured_docker
+db:
+    image: kartoza/postgis:11.0-2.5
+    volumes:
+      - ./ssl_cert.pem:/configs/ssl_cert.pem
+      - ./ssl_key.pem:/configs/ssl_key.pem
     environment:
-      - SSL_CERT_FILE=/etc/ssl/certs/ssl_cert.pem
-      - SSL_KEY_FILE=/etc/ssl/private/ssl_key.pem
+      # If you need to create multiple database you can add coma separated databases eg gis,data
+      - POSTGRES_DB=gis
+      - POSTGRES_USER=docker
+      - POSTGRES_PASS=docker
```

Comment:
I deliberately skipped those vars in this example to keep it focussed on what they should demonstrate.

Totally unrelated: from a security perspective I think providing login credentials through env vars is not optimal (but can't come up with a better solution right now).

Reply:
I have addressed this